packages/sdk-browser/src/sdk-client.ts

```diff
@@ -246,8 +252,145 @@ export class TurnkeyPasskeyClient extends TurnkeyBrowserClient {
       attestation: attestation,
     };
   };
+
+  createPasskeySession = async (
```

Comment:
Important to note that this won't be compatible with rn unless https://www.npmjs.com/package/react-native-quick-crypto gets to a place where all this polyfill is supported. Potentially out of this scope, but as some point we prob want to put all of this encoding/decoding logic in @turnkey/crypto

Reply:
Definitely. I'm currently not expecting this to be used in the RN context. I think it's worth picking up/prioritizing later on if there's an explicit ask , at which point we'd probably need to... essentially implement hpke encrypt 👀